test/integration/project_index_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+
+class ProjectIndexTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+  include FactoryGirl::Syntax::Methods
+
+  test "pagination renders on page" do
+		15.times do
+			create(:project)
+		end
+
+		visit projects_path
+
+		assert page.has_content?("Next")
+		assert page.has_content?("Previous")
+	end
```

Comment:
`end` at 16, 1 is not aligned with `test "pagination renders on page" do` at 7, 2<br>Tab detected.